Scripts/RemediationScripts/Remediate-EnableHttpsForStorageAccount.ps1

```diff
@@ -0,0 +1,723 @@
+﻿<##########################################
+
+# Overview:
+    This script is used to make the transit in Storage Account encrpyted using the HTTPS.
+
+# ControlId: 
+    Azure_Storage_DP_Encrypt_In_Transit
+
+# DisplayName:
+    Enable Secure transfer to Storage Accounts
+
+# Pre-requisites:
+    You will need atleast contributor role on Storage Account(s) of subscription.
+
+# Steps performed by the script :
+   To Remediate:
+        1. Checking for prerequisites.
+        2. Validating the Account type.
+        3. Removing   the excluded resource group(s) and resource  from the list of subscription.
+        4. Get list of Storage Account in a subscription which are having 'EnableHttpsTrafficOnly' value set to false.
+        5. Taking backup of Storage Account(s) having value of 'EnableHttpsTrafficOnly' set to false that are going to be remediated using remediation script.
+        5. Set the value of 'EnableHttpsTrafficOnly' to true for the Storage Account(s) of subscription.
+
+   To rollback:
+        1. Validate and install the modules required to run the script.
+        2. Get the list of Storage Accounts in a Subscription, the changes made to which previously, are to be rolled back.
+        3. Set the value of 'EnableHttpsTrafficOnly'  to 'false' on the Storage Account in the Subscription.
+
+# Instructions to execute the script:
+    To remediate:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to enable "EnableHTTPSTrafficOnly" on the Storage Account in the Subscription. Refer `Examples`, below.
+   
+    To roll back:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to disable "EnableHTTPSTrafficOnly"  on the Storage Account in the Subscription. Refer `Examples`, below.
+
+# Examples:
+    To remediate:
+        1. To review the Storage Account in a Subscription that will be remediated:
+           Enable-StorageEncryptionInTransit  -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000  -DryRun
+
+        2. To enable "EnableHTTPSTrafficOnly"  on the Storage Account in a Subscription:
```

Comment:
`EnableHTTPSTrafficOnly` - This property name may not make immediate sense to a user who is looking to use this script. This can be replaced with something like: "To enable encryption in transit..."
This needs to be done at multiple places.

Reply:
Done